terraform/deployments/terraform-lock/README.md

```diff
@@ -18,11 +18,11 @@ Terraform docs for more detail.
 ## Applying
 
 ```sh
-terraform init -backend-config=<environment>.backend -upgrade
-terraform apply
+ENV=test
```

Comment:
Does this advise folks to use the test env dynamodb for locking in all envs?

Reply:
Fixed